kolibri/plugins/learn/assets/src/views/content-cards/content-carousel.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+
+  <section class="content-carousel">
+
+    <header v-if="header" class="content-carousel-header">
+      <h2 v-if="header"> {{header}} </h2>
+      <sub v-if="subheader"> {{subheader}} </sub>
+    </header>
+
+    <div class="content-carousel-set">
+      <transition :name="animation">
+        <div :key="currentSetIndex" class="content-carousel-cards" :style="widthOfCarousel">
+          <content-card
+          v-for="content in contentSets[currentSetIndex]"
+          class="content-card"
+          :title="content.title"
+          :thumbnail="content.thumbnail"
+          :kind="content.kind"
+          :progress="content.progress"
+          :link="genContentLink(content.id)"/>
+        </div>
+      </transition>
+
+      <div class="content-carousel-controls">
+        <icon-button class="previous" @click="previousSet" v-if="!isFirstSet">
```

Comment:
[Keen UI's icon-button](https://josephuspaye.github.io/Keen-UI/#/ui-icon-button) might be best here. But ya I know it's not wrapped yet.